src/Playwright.Tests/PageEventNetworkTests.cs

```diff
@@ -130,14 +130,14 @@ public async Task ShouldSupportRedirects()
             string FOO_URL = Server.Prefix + "/foo.html";
             var response = await Page.GotoAsync(FOO_URL);
             await response.FinishedAsync();
-            Assert.AreEqual(new[] {
+            CollectionAssert.AreEquivalent(new[] {
```

Comment:
what is there the difference to Assert.AreEqual?

Reply:
`AreEqual` assumes order is the same, while `AreEquivalent` ensures all items are there, but can be in different orders.

Reply:
We want to have AreEqual in most cases, otherwise we use Sets upstream.